src/Impostor.Server/Net/Inner/Objects/InnerPlayerInfo.cs

```diff
@@ -37,6 +38,13 @@ public InnerPlayerInfo(byte playerId)
 
         public List<InnerGameData.TaskInfo> Tasks { get; internal set; }
 
+        public DateTimeOffset LastMurder { get; set; }
+
+        public bool CanMurder(IGame game)
+        {
+            return DateTimeOffset.UtcNow.Subtract(LastMurder).TotalSeconds < game.Options.KillCooldown;
+        }
```

Comment:
This is a public method which returns the exact opposite of what it should return. Flip this with `>=`.

Reply:
Whoops